doc/programming_guide/resource_handlers.rst

```diff
@@ -1,24 +1,24 @@
-.. _resource_handlers:
+.. _resource_handles:
 
-Adding Your Own Resource Handlers
-=================================
+Adding Your Own Resource Handles
+================================
 
-Arcade provides a convenient way to locate asset through it's resource handlers. Arcade already has a number of
-Built-In sprites, images and other resources available for use inside the ``:resources:`` handler, which you
-may already be familiar with:
+Arcade provides a convenient way to locate asset through it's resource handles. Arcade already has a number of
+Built-In sprites, images and other resources available for use inside the ``:assert::``, and ``:system:``
```

Comment:
Typo:
```suggestion
Built-In sprites, images and other resources available for use inside the ``:assets:``, and ``:system:``
```

Reply:
I'll get it in the resources vs assets pass.